### PR TITLE
add kafka instance view.

### PR DESCRIPTION
**Pull request by Lico-Tom**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { KubernetesInstancesComponent } from "./kubernetes/kubernetes-instances/kubernetes-instances.component";
+import { KafkaInstancesComponent } from "./kafka/kafka-instances/kafka-instances.component";
 import { HomePageComponent } from "./home-page/home-page.component";
-
+// 模块添加路由
```
Comment (Shoothzj) [inline src/app/app-routing.module.ts:6]: please remove this line

File: src/app/app.module.ts [hunk 2]
```diff
@@ -10,12 +10,14 @@ import { HomePageComponent } from './home-page/home-page.component';
...
 import { MatTableModule } from "@angular/material/table";
 import { HttpClientModule } from "@angular/common/http";
+import { KafkaInstancesComponent } from './kafka/kafka-instances/kafka-instances.component';
 
 @NgModule({
   declarations: [
     AppComponent,
     HomePageComponent,
     KubernetesInstancesComponent,
+    KafkaInstancesComponent,
```
Comment (Shoothzj) [inline src/app/app.module.ts:20]: please sort alphabet and format the import

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { KubernetesInstancesComponent } from "./kubernetes/kubernetes-instances/kubernetes-instances.component";
+import { KafkaInstancesComponent } from "./kafka/kafka-instances/kafka-instances.component";
```
Comment (Shoothzj) [inline src/app/app-routing.module.ts:4]: please sort alphabet and format the import

**Review by Shoothzj — APPROVED** [review]

File: src/app/app-routing.module.ts (hunk 3 above)
Comment (Lico-Tom) [reply src/app/app-routing.module.ts:3]: ok

**Pull request merged**